src/Raven.Server/Smuggler/Documents/DatabaseSource.cs

```diff
@@ -602,5 +603,89 @@ public SmugglerSourceType GetSourceType()
         {
             return _type;
         }
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
write a sync version (the one below) and do `ToAsyncEnumerable()`

Reply:
fixed, thanks